### DISCUSSION TITLE
Licence doesnt work

### DISCUSSION THREAD

**Discussion by Larovich** [aclap-dev/video-downloadhelper#426] [Help and Q&A]
### Your question

I enter the license on the site, and it endlessly checks it, what should I do?

### Addon & CoApp version and details

![Снимок экрана (605)](https://github.com/aclap-dev/video-downloadhelper/assets/168019811/14e7c8dc-a401-43df-b0f8-29f4408c41af)

**Comment by Larovich** [comment]
this photo*
![Снимок экрана (608)](https://github.com/aclap-dev/video-downloadhelper/assets/168019811/3180aef0-35a1-4370-9a77-b5cc227891f9)

**Reply by Larovich** [reply]
last line

**Comment by jcfrog** [comment]
hi, might be an antivirus or firewall issue. Check the CoApp is not blocked.
